packages/twenty-front/src/modules/settings/data-model/fields/forms/select/components/SettingsDataModelFieldSelectFormOptionRow.tsx

```diff
@@ -133,6 +134,7 @@ export const SettingsDataModelFieldSelectFormOptionRow = ({
           })
         }
         RightIcon={isDefault ? IconCheck : undefined}
+        maxLength={NAME_MAXIMUM_LENGTH}
```

Comment:
I feel NameMaximumLength is not a proper name for select options that have either label or values. We should either create another constant OptionLabelMaxLength or use a more generic name such as the one use by postgres: DatabaseIdentiferMaxLength